### PR TITLE
UI Page [Settings] [Check Usage]

### PR DESCRIPTION
**Pull request by H0llyW00dzZ**
Ref : https://github.com/Yidadaa/ChatGPT-Next-Web/discussions/2936

Note :
- For use a sess-key in web version try change to "https://api.openai.com/" instead of "/api/openai/" for the endpoint
- In Client-App Work with "/api/proxy" example : https://chatgpt.btz.pm/api/proxy Also with "sess-key" if you already made api key it work's to interaction with all models there

[+] fix(settings): change showUsage condition from accessStore.isAuthorized() to accessStore.token.startsWith("sess-")
[+] chore(locales): update No Access for check usages messages in multiple languages

### CODE REVIEW

**Comment by H0llyW00dzZ** [comment]
Preview In client : 

image

File: app/components/settings.tsx [hunk 1]
```diff
@@ -599,7 +599,7 @@ export function Settings() {
   const customCount = promptStore.getUserPrompts().length ?? 0;
   const [shouldShowPromptModal, setShowPromptModal] = useState(false);
 
-  const showUsage = accessStore.isAuthorized();
+  const showUsage = accessStore.token.startsWith("sess-");
```
Comment (H0llyW00dzZ) [inline app/components/settings.tsx:602]: by filtering like this as prefix because the logic we already `Authorized`

**Comment by Yidadaa** [comment]
sess- key is bad, we dont need this.

**Comment by H0llyW00dzZ** [comment]
@Yidadaa account balance for check it only works with sess-key

**Comment by Yidadaa** [comment]
I know, this is an outdated feature, I keep it now just because some third party openai forwarding service also provides balance query interface, that's it, it will be removed in the future.

**Comment by H0llyW00dzZ** [comment]
oh okay

**Pull request closed without merging**